resource-server/okta-aspnet-webapi-example/Web.config

```diff
@@ -6,231 +6,231 @@
 <configuration>
   <appSettings>
     <!-- 1. Replace these values with your Okta configuration -->
```

Comment:
Maybe we should change to `Replace these value with your Okta configuration` singular "value" since there's one one value now